skimage/util/dtype.py

```diff
@@ -292,9 +292,9 @@ def _scale(a, n, m, copy=True):
             # if imin_in:
             #     np.maximum(image, -1.0, out=image)
         else:
-            image = np.multiply(image, 2. / (imax_in - imin_in),
-                                dtype=computation_type)
-            image += 1.0 / (imax_in - imin_in)
+            image = np.add(image, 0.5, dtype=computation_type)
```

Comment:
Did this really lead to an inacurracy?

Interesting.

Reply:
Yes, it did. See the `np.int8` test case that @stefanv added. It fails without this change, due to floating point arithmetic error.